tox.ini

```diff
@@ -0,0 +1,40 @@
+[tox]
+minversion = 3.9.0
+envlist =
+    lint
+    py{36,37,38}
+
+# do not enable skip missing to avoid CI false positives
+skip_missing_interpreters = False
+isolated_build = True
+
+[testenv]
+# develop temporary disabled as project packaging does not work with it yet:
+# https://github.com/willmcgugan/rich/issues/345
+usedevelop = False
+# do not put * in passenv as it may break builds do to reduced isolation
+deps =
+    -rrequirements-dev.txt
```

Comment:
Should there not be a space after `-r` ?

Reply:
Not needed, pip works fine with both. I used that because on openstack projects is seems to be the practice, but I am happy to add the space if you want.